src/Domain/Person.php

```diff
@@ -111,7 +142,11 @@ class Person implements JsonSerializable
      */
     public ?string $raw_password = null;
 
-    private ?string $stripe_customer_id = null;
+    /**
+     * @ORM\Column(type="string", nullable=true)
+     * @OA\Property()
+     */
+    public ?string $stripe_customer_id = null;
```

Comment:
@NoelLH Just a point of thought... is it ok to keep these member variables `public` and access them directly? Or should we make everything `private` and access them via their getters? I'll let you decide.

Reply:
@alihejazi I decided to try this route because with the Symfony Serializer library it hugely reduces the boilerplate and total amount of code. Compare all the separate HTTP models that MatchBot needs to accomplish the equivalent.